assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -124,12 +125,18 @@ const ChatChannel = RestModel.extend({
 
     return this.memberships_count;
   },
+
+  @computed("current_user_membership.following")
+  get isFollowing() {
```

Comment:
Not sure if it is worth also making shortcuts like `isMuted` for these types of things, `isFollowing` was just the most used

Reply:
Yep I'm fine just adding them later as the need arises